ci/jobs/satellite6-pulp.yaml

```diff
@@ -0,0 +1,187 @@
+# This job automates the process of deploying Satellite 6 on OpenStack instances
+# and upgrading the Pulp for Robottelo and Pulp Smash tests.
+# Prerequisites:
+#     1) Activate account on OpenStack platform.
+#     2) Download the OpenStack RC file from Access&Security>API Access.
+#     3) Create a secret credential file <os-secret.sh> with the following format:
+#         export OS_AUTH_URL=OS_AUTH_URL
+#         export OS_USERNAME=OS_USERNAME
+#         export OS_PASSWORD=password
+#         export OS_TENANT_NAME='"US.ENG Perf R&D"'
+#     4) Upload this secret file to Jenkins UI, for a credentials-binding id.
+#     5) Create a secret credential file <jenkins-secret.sh>:
+#         export REMOTE_JENKINS_USERNAME=username
+#         export REMOTE_JENKINS_API_TOKEN=token
+#         export REMOTE_JENKINS_URL=<remote-jenkins-server-host>
+#     4) Upload this secret file to Jenkins UI, for a credentials-binding id.
+# The automation steps are as following:
+#     1) prebuild to create OS1 instances of RHEL 7.2;
+#     2) run external Jenkins Satellite6-installer to deploy 6.2 Downstream;
+#     3) run pulp-upgrading script to update Pulp from 2.8.3 to 2.8.z;
+#     4) run external Robottelo-runner and internal pulp-smash-runner.
+# This Jenkins job thus glues these steps together, in order to provide automated sanity checks.
+
+- job-template:
+    name: "{name}"
+    node: f23-vanilla-np
+    properties:
+        - qe-ownership
+    wrappers:
+        - ansicolor
+        - ssh-agent-credentials:
+            users:
+                - '044c0620-d67e-4172-9814-dc49e443e7b6'
+        - credentials-binding:
+            - file:
+                credential-id: 7ca95790-1c85-4418-ac4c-082a366cf2db
+                variable: REMOTE_JENKINS_CREDENTIALS
+            - file:
+                credential-id: ff1c87ec-76f1-4bb1-a947-fd2114cd8fd9
+                variable: OPENSTACK_CREDENTIALS
+    builders:
+        - shell: |
+            source $OPENSTACK_CREDENTIALS
+            source $REMOTE_JENKINS_CREDENTIALS
+
+            echo "Preparing OpenStack client environment on Jenkins server..."
+            sudo yum -y update python
+            sudo yum -y install wget
+            sudo pip install pip --upgrade
+            sudo pip install virtualenv --upgrade
+            virtualenv sat-pulp
+            source sat-pulp/bin/activate
+            pip install pbr
```

Comment:
Hi folks, I have dropped that line and it works...thank you for your notes on it!